autogluon/autogluon-cloud
# Use latest AMI for cluster

Issue #, if available:

Description of changes:
* Add util to fetch latest ami
* Add custom ami support in config
* Remove some duplicate code in config generator

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
